spyder/preferences/languageserver.py

```diff
@@ -608,30 +610,70 @@ def mouseDoubleClickEvent(self, event):
 class LSPManagerConfigPage(GeneralConfigPage):
     """Language Server Protocol manager preferences."""
     CONF_SECTION = 'lsp-server'
-    NAME = _('Language Server Protocol')
+    NAME = _('Code completion and linting')
     ICON = ima.icon('lspserver')
 
     def setup_page(self):
+        # --- Other servers tab ---
+        # Section label
+        servers_label = QLabel(
+            _("Spyder uses the <a href=\"{lsp_url}\">Language Server "
+              "Protocol</a> technology to provide code completion and linting "
+              "for its Editor. Here you can define how to start a server for "
+              "other programming languages besides Python, along with its "
+              "respective configuration. After doing that, Spyder will be "
+              "able to give completions and linting for the files associated "
+              "with that language."
```

Comment:
``Here you can [...] associated with that language`` -> ``Here, you can setup and configure LSP servers for languages other than Python, so Spyder can provide such features for those languages as well.``